build-system/tasks/bundle-size/index.js

```diff
@@ -170,7 +166,7 @@ async function skipBundleSize() {
  */
 async function reportBundleSize() {
   if (isPullRequestBuild()) {
-    const baseSha = gitCiMasterBaseline();
```

Comment:
I think the change in this PR should apply to both places that call this function (bundle size and visual diff). The intent behind this function is to provide a baseline for diffs, and I believe they should be identical. Perhaps reimplement the original function instead of adding a new one? WDYT?

Reply:
Outdated, since we're just persisting the merge commit now.